### PR TITLE
linking issue resolved.

### PR DESCRIPTION
**Pull request by abubaker47**
Hoopoe books resources published in multiple languages are unable to be joined. Issue https://github.com/ddlibrary/website/issues/160 Fixed.

### CODE REVIEW

File: app/Http/Controllers/ResourceController.php [hunk 1]
```diff
@@ -63,9 +63,13 @@ public function index(Request $request)
 
     public function updateTid(Request $request, $resourceId)
     {
-        $resource = Resource::findOrFail($request->input('link')); 
-        $resource->tnid = $resourceId;
-        $resource->save();  
+        $translatedResource = Resource::findOrFail($request->input('link')); 
+        if($translatedResource)
```
Comment (uyscuti-wiki) [inline app/Http/Controllers/ResourceController.php:67]: Question: isn't the `if` sort of redundant? if there's no `$translatedResource`, shouldn't `findOrFail` cause the script to fail?

File: config/laravellocalization.php [hunk 2]
```diff
@@ -242,12 +242,12 @@
...
-        'sw'          => ['name' => 'Savji',                    'script' => 'Arab', 'native' => 'سوجی', 'regional' => 'fa_IR'],
-        'sh'          => ['name' => 'Sheghnani',                'script' => 'Arab', 'native' => 'شغنانی', 'regional' => 'fa_IR'],
-        'pa'          => ['name' => 'Pashai',                   'script' => 'Arab', 'native' => 'پشه یی', 'regional' => 'fa_IR'],
+        'uz'          => ['name' => 'Uzbeki',                   'script' => 'Arab', 'native' => 'ازبیکی', 'regional' => 'uz_AF'],
+        'mj'          => ['name' => 'Munji',                    'script' => 'Arab', 'native' => 'منجی', 'regional' => 'mj_AF'],
+        'no'          => ['name' => 'Nooristani',               'script' => 'Arab', 'native' => 'نورستانی', 'regional' => 'no_AF'],
+        'sw'          => ['name' => 'Savji',                    'script' => 'Arab', 'native' => 'سوجی', 'regional' => 'sw_AF'],
+        'sh'          => ['name' => 'Sheghnani',                'script' => 'Arab', 'native' => 'شغنانی', 'regional' => 'sh_AF'],
+        'pa'          => ['name' => 'Pashai',                   'script' => 'Arab', 'native' => 'پشه یی', 'regional' => 'pa_AF'],
```
Comment (uyscuti-wiki) [inline config/laravellocalization.php:250]: don't fully understand why we had to make this change. Could you explain Faisal?

**Review by uyscuti-wiki — COMMENTED** [review]
Great work Faisal, thanks! Couple comments inline.

File: app/Http/Controllers/ResourceController.php (hunk 1 above)
Comment (abubaker47) [reply app/Http/Controllers/ResourceController.php]: Thanks for addressing this, I have removed the redundant IF statement in the new commit.

File: config/laravellocalization.php (hunk 2 above)
Comment (abubaker47) [reply config/laravellocalization.php:250]: This wasn't necessary but the language region was Iran and I changed it to Afghanistan as these languages region is Afghanistan.

**Pull request merged**